ui/mainframe.py

```diff
@@ -154,23 +154,23 @@ def __init__(self, mainUI=None):
         self.takeBackupBtn.configure(highlightcolor="black")
         self.takeBackupBtn.configure(pady="0")
         self.takeBackupBtn.configure(text='''Take Backup''')
-
-        userVar = tk.StringVar(mainUI, "User")
-        adminVar = tk.StringVar(mainUI, "Admin")
+        self.userVar = tk.StringVar(mainUI, "User")
+        self.adminVar = tk.StringVar(mainUI, "Admin")
         self.style.map('TRadiobutton', background=[('selected', _bgcolor), ('active', _ana2color)])
         self.radioAdmin = ttk.Radiobutton(mainUI)
         self.radioAdmin.place(relx=0.65, rely=0.375, relwidth=0.091
                               , relheight=0.0, height=26)
         self.radioAdmin.configure(text='''Admin''')
+        self.radioAdmin.configure(variable=self.adminVar)
         self.radioAdmin.configure(value=1)
```

Comment:
Can this value be a String instead of a number?
If yes, please change it to a meaningful value like `user` and `admin`

Reply:
for admin value will be "admin"
use="user"

Reply:
perfect

Reply:
value changed but with name user_type_admin
user_type_user